File: easy/1-Two_Sum.cpp
vector<int> twoSum(vector<int>& nums, int target) {
	unordered_map<int, int> m;
	vector<int> ans;
	
	for(int a = 0; a < nums.size(); a++) {
		if(m.count(target-nums[a]) == 1) {
			ans.push_back(a);
			ans.push_back(m.at(target-nums[a]));
			break;
		}
		
		m[nums[a]] = a;
		
	}
	
	return ans;
}

File: easy/1-Two_Sum.java
public int[] twoSum(int[] nums, int target) {
	HashMap<Integer, Integer> m = new HashMap<>();
	int[] ans = new int[2];
	
	for(int a = 0; a < nums.length; a++) {
		if(m.containsKey(new Integer(target-nums[a]))) {
			ans[0] = a;
			ans[1] = m.get(new Integer(target-nums[a])).intValue();
			break;
		}
		
		m.put(new Integer(nums[a]), new Integer(a));
	}
	
	return ans;
		
}